modules/api/pkg/api/v1/types.go

```diff
@@ -1019,6 +1019,12 @@ type ClusterSpec struct {
 	// By default it is disabled.
 	OPAIntegration *kubermaticv1.OPAIntegrationSettings `json:"opaIntegration,omitempty"`
 
+	// A map of optional or early-stage features that can be enabled for the user cluster.
+	// Some feature gates cannot be disabled after being enabled.
+	// The available feature gates vary based on KKP version, Kubernetes version and Seed configuration.
+	// Please consult the KKP documentation for specific feature gates.
+	Features map[string]bool `json:"features,omitempty"`
```

Comment:
If I understand it correctly what this line does is that the features field can now also be set through the api.
If that is the case and you would need to set it through KKP api, then wouldn't there be an equivalent for "ClusterBackUpStorage" in the list from here: https://github.com/kubermatic/kubermatic/blob/main/pkg/apis/kubermatic/v1/cluster.go#L325

But I could be wrong, I think someone from sig-cluster-mgmt needs to take a look at this. Or maybe to ask the other way around: What feature will be set through the api with this change?

Reply:
Actually, this needs to be removed. It was added before because of another approach we wanted to use, but right now, we don't want it. I'll delete it.